### ISSUE TITLE
Machine learning algorithm

### ISSUE THREAD

**Issue by 9il** [libmir/dcv#17]
Does existing or future dcv requires ML algorithms? If yes, please open an issue in https://github.com/libmir/mir/issues . Most of ML can be split to data preparation and other computations. _Other_ computations_ may be added to Mir as building blocks for ML algorithms in dcv.

**Comment by ljubobratovicrelja** [comment]
> Does existing or future dcv requires ML algorithms?

Yes! I'll look into it, and let you know via issue in Mir. Thanks!

**Comment by 9il** [comment]
Please include matrix and vector algorithms. For example, I saw dcv uses scid for matrix inversion. This should be included to the Mir after a while
